src/strategies/unigraph/index.ts

```diff
@@ -0,0 +1,66 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+const UNISWAP_SUBGRAPH_URL = {
+  1: 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2',
```

Comment:
Can you map others Uniswap subgraphs here? Also some new fields are missing (version and github author) see example here: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/balancer/index.ts#L4-L5

Reply:
I wasn't able to find an official testnet subgraph like balancer has, wasn't sure whether to use a community one.